admin/installer/_images.py

```diff
@@ -67,3 +120,269 @@ def _unserialize_packer_dict(serialized_packer_dict):
         key, value = item.split(":")
         packer_dict[key] = value
     return freeze(packer_dict)
+
+
+class PackerConfigure(PClass):
+    """
+    The attributes necessary to create a custom packer configuration file from
+    the prototype files in ``admin/installer/packer``.
+
+    :ivar build_region: The AWS region to build images in.
+    :ivar publish_regions: The AWS regions to publish the build images to.
+    :ivar template: The prototype configuration to use as a base. One of
+        `docker` or `flocker`.
+    :ivar distribution: The operating system distribution to install.
+        ubuntu-14.04 is the only one implemented so far.
+    :ivar configuration_directory: The directory containing prototype
+        configuration templates.
+    :ivar source_ami: The AMI ID to use as the base image.
+    :ivar working_directory: The directory in which templates will be copied
+        and modified.
+    """
+    build_region = field(type=unicode, mandatory=True)
+    publish_regions = pvector_field(item_type=unicode)
+    template = field(type=unicode, mandatory=True)
+    distribution = field(type=unicode, mandatory=True)
```

Comment:
It helped a lot with the up front validation of build_region, 
I ditched `publish_regions` since I don't think I'll need that yet.
I looked for some existing "supported distributions" constants...there some stuff in admin/packaging  which I can reuse / refactor in a followup.